PelotonEppSdkTests/TestBase.cs

```diff
@@ -10,8 +10,8 @@ public class TestBase
     {
         protected TestBase()
         {
-            baseUri = new Uri("https://testapi.peloton-technologies.com");
-            //baseUri = new Uri("http://localhost:2590/api");
+            //baseUri = new Uri("https://testapi.peloton-technologies.com");
+            baseUri = new Uri("http://localhost:2590/");
```

Comment:
Please revert this change. We want the baseuri to point to testapi.peloton-technologies.com.